common/src/main/java/com/wynntils/core/config/objects/ConfigHolder.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.config.objects;
+
+import com.wynntils.core.config.properties.Config;
+import com.wynntils.core.features.properties.FeatureInfo;
+import java.lang.reflect.Field;
+
+public class ConfigHolder extends StorageHolder {
+    private final Config metadata;
+
+    public ConfigHolder(Object parent, Field field, Config metadata) {
+        super(
+                parent,
+                field,
+                field.getType(),
+                field.getAnnotation(FeatureInfo.class).category(),
+                metadata.visible());
+
+        this.metadata = metadata;
+        // save default value to enable easy resetting
```

Comment:
Is this a to-do? If so, could you mark it `TODO: ..`.

Reply:
No that was there before, I think P0ke added it, I just left it there. It can be removed if you want

Reply:
Yeah, if it does not mean anything in the new context, please remove it, so it's not confusing.

Reply:
alternatively, move it to the line in `StorageHolder` where the default value is being saved now